topomodelx/nn/hypergraph/hypersage_layer.py

```diff
@@ -51,6 +51,8 @@ class HyperSAGELayer(MessagePassing):
         Dimension of the input features.
     out_channels : int
         Dimension of the output features.
+    alpha: int, default=-1
```

Comment:
NIT: there is a space before the ":" in numpy's docstring conventions (which are the ones we use in the library) for some reason.